fairlearn/metrics/_metric_frame.py

```diff
@@ -438,7 +529,10 @@ def sensitive_levels(self) -> List[str]:
         return self._sf_names
 
     def __group(
```

Comment:
hadn't noticed we have something starting with `__`, instead of `_`, why is that?

Reply:
I don't remember. It might have been because this was considered an _extra_ private method. \Bbut that's not part of this PR :-)